src/File/FileManagerInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\File;
+
+interface FileManagerInterface
+{
+    public function createWithContent(string $fileName, string $content): void;
+
+    public function remove(string $fileName): void;
+
+    public function getBaseDirectory(): string;
```

Comment:
This leaks some implementation logic (the concept of a base directory, where files are stored directly under). What do you think about changing it to `public function realpath(string $fileName): string`, which would return an absolute, canonicalised path to the file?